### DISCUSSION TITLE
Is it possible to customize the frontend after the deploy has been done through Azure?

### DISCUSSION THREAD

**Discussion by leonardobora** [Azure-Samples/chat-with-your-data-solution-accelerator#1065] [Q&A]
Hey there everyone. Thanks for the awesome repository. I have found this solution to be exactly what I need for my business' purpose. 

Although I am happy with the overall responses of the model after setting up RAG with 100+ documents, I would like to customize the front-end to meet my business brand schema. 

Since I haven't deployed it locally, am I able to re-deploy only the part of the code responsible for the design? From what I have looked, everything seems to be fixed in the code. Am I wrong?

Help would be much appreciated :)

Thanks in advance!

**Comment by ellacully** [comment]
Hi! 

I was also curious about this, was wondering if you were able to make any progress? Thank you!

**Comment by Roopan-Microsoft** [comment]
Hey @leonardobora - Thanks for reaching out to us.

May I know what actually you are trying to customize in the frontend. Please provide some insights.

Its actually container based, you can deploy your own container and publish yours.

Thanks for your patience.

Happy to Help!

**Reply by Celiuqui8** [reply]
Hey!
I personally would like to be able to change the name, icons and images to make it more personal.
Thanks!

**Reply by Roopan-Microsoft** [reply]
Hey yeah, you can do that. 

You can do the required code changes and publish it to the container image. 

During the deployment, you need to map that container to your code like below, you need to replace below highlighted to your container. [Check for other occurrences as well]

<img width="682" alt="image" src="https://github.com/user-attachments/assets/36d2881d-f538-4424-b57e-adfd755120a1">

Please let me know if any help required.

Happy to Help!

**Comment by lucamazzasette** [comment]
I too have been interested in the project and have installed the solution via Deploy on Azure on the project page.
I wonder, since I am not a programmer, how can I customize the page by changing the logo and slogan on the front page.
Is it possible?

**Reply by Roopan-Microsoft** [reply]
Hey @lucamazzasette - Thanks for showing interest in this.

Ideally, we don’t include that customization in this accelerator. However, you can achieve it by making the necessary changes locally, publishing to a different container, and pointing that to your app service.

Please let us know if any help required.

**Reply by lucamazzasette** [reply]
On Azure i don't find the container, I see only the webapp and into it don't exist any reference to change logo and favicon. 
While in the git project , after clone it in local, I can find and change but non exist the file.sln for publish that.
I'm not Developer and I need a  support for using this fantastic project. 
Please, if possible, give me step by step for deploy the local clone repository  into webapp for reach the goal.
Thanks
